src/sentry/options/defaults.py

```diff
@@ -2735,6 +2735,14 @@
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Compression level for spans buffer segments. -1 disables compression, 0-22 for zstd levels
+register(
+    "spans.buffer.compression.level",
+    type=Int,
+    default=0,
```

Comment:
Let's set this to off by default for now, so we can enable it during runtime.

Reply:
done